.github/CODEOWNERS

```diff
@@ -0,0 +1,42 @@
+.github/                        @dart-ecosystem-admins
+README.md                       @dart-ecosystem-admins
+pkgs/bazel_worker               @dart-bat #create
+pkgs/benchmark_harness          @dart-vm #create
+pkgs/boolean_selector           @dart-ecosystem-admins #assign
+pkgs/browser_launcher           @dart-ecosystem-admins #assign
+pkgs/cli_config                 @dcharkes
+pkgs/cli_util                   @devoncarew
+pkgs/clock                      @dart-ecosystem-admins #assign
+pkgs/code_builder               @dart-ecosystem-admins #assign
+pkgs/coverage                   @liamappelbe @dart-vm
+pkgs/csslib                     @leonsenft
```

Comment:
I'd like to request I _not_ be added as the owner here please. I spent some time years ago fixing issues that affected ACX Framework and its users, but we recently dropped this dependency (the main motivation being that this package is largely unmaintained and out of date with modern CSS grammar).